Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.23.2-alpine
+
+COPY . /app
+
+WORKDIR /app
+
+RUN /app/install-task.sh -b /usr/local/bin
```

Comment:
My main gripe with this is that is is, at least as far as I understand, quite impossible for docker to cache any layers to speed up builds.